### ISSUE TITLE
Using reserved words as column name

### ISSUE THREAD

**Issue by hamishjohnson** [supabase/postgrest-js#432]
This has already been reported and closed elsewhere

https://github.com/supabase/supabase/issues/12539

The solution was to update documentation but I think, if possible, a better solution would be to add this to the runtime error message, as it's not easy or intuitive to find in the documentation, and if it was directly in the error message I would have easily avoided many hours of troubleshooting.

**Comment by steve-chavez** [comment]
Q: Is your column named "order"?

Regardless of whether we can err(which I'm not opposed to), that could be trouble when using it inside a SQL query. You would have to quote "order" everywhere.

**Comment by hamishjohnson** [comment]
Yeah, exactly - I was using order. The point of this PR is just that I couldn't understand why for a long time, and the documentation is not clear on it. I think an error message would make this a lot clearer.
